shared/js/background/message-handlers.js

```diff
@@ -154,44 +154,10 @@ export function getTopBlockedByPages (options) {
     return Companies.getTopBlockedByPages(options)
 }
 
-// Click to load interactions
-export async function initClickToLoad (config, sender) {
+export async function getClickToLoadClicks () {
     await settings.ready()
-    const tab = tabManager.get({ tabId: sender.tab.id })
-
-    // Remove first-party entries.
-    await startup.ready()
-
-    // Overwrite the content-scope-scripts Click to Load configuration with the
-    // legacy configuration for the integration tests.
-    // TODO: When the legacy configuration is finally removed, this should be
-    //       adjusted.
-    if (await getDevMode() && await getIntegrationTestMode()) {
-        config = tdsStorage.ClickToLoadConfig
-    }
-
-    const siteUrlSplit = tab.site.domain.split('.')
-    const websiteOwner = trackers.findWebsiteOwner({ siteUrlSplit })
-    if (websiteOwner) {
-        delete config[websiteOwner]
-    }
-
-    // Determine whether to show one time messages or simplified messages
-    for (const entity of Object.keys(config)) {
-        const clickToLoadClicks = settings.getSetting('clickToLoadClicks') || {}
-        const maxClicks = config[entity].clicksBeforeSimpleVersion || 3
-        if (clickToLoadClicks[entity] && clickToLoadClicks[entity] >= maxClicks) {
-            config[entity].simpleVersion = true
-        }
-    }
-
-    // if the current site is on the social exception list, remove it from the config.
-    let excludedNetworks = trackerutils.getDomainsToExludeByNetwork()
-    if (excludedNetworks) {
```

Comment:
Yep, sorry I should have mentioned that. The content-scope-script repo takes care to only init features that are enabled for the current page, so we don't need to worry about the domain exceptions in [click-to-play.json](https://github.com/duckduckgo/privacy-configuration/blob/main/features/click-to-play.json) here (or to worry about checking that the feature is enabled).

We also don't need to worry about `trackerutils.getDomainsToExludeByNetwork()` here since 1. that is a check for [excludedDomains in the legacy config](https://staticcdn.duckduckgo.com/useragents/social_ctp_configuration.json) which is so far empty and 2. that config and corresponding logic will all be going away shortly!